### ISSUE TITLE
Basic Use of "ls" Breaking

### ISSUE THREAD

**Issue by dprice** [PowerShell/GraphicalTools#31]
I tried this basic command testing the module: `ls | ogv`

- Windows 5.1:
Works

- Windows Core 6.x & 
```
ogv : Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index
At line:1 char:6
+ ls | ogv
+      ~~~
+ CategoryInfo          : NotSpecified: (:) [Out-GridView], ArgumentOutOfRangeException
+ FullyQualifiedErrorId : System.ArgumentOutOfRangeException,OutGridView.Cmdlet.OutGridViewCmdletCommand
```

Any ideas?

Thanks,
Derek

**Comment by TylerLeonhardt** [comment]
@dprice what version of GraphicalTools are you using? (`Get-Module -list GraphicalTools`)

**Comment by dprice** [comment]
> @dprice what version of GraphicalTools are you using? (`Get-Module -list GraphicalTools`)

```
C:\Users\derek.price> Get-Module -list Microsoft.PowerShell.GraphicalTools


    Directory: C:\Users\derek.price\Documents\PowerShell\Modules

ModuleType Version    PreRelease Name                                PSEdition ExportedCommands
---------- -------    ---------- ----                                --------- ----------------
Script     0.1.1                 Microsoft.PowerShell.GraphicalTools Core      {Out-GridView, ogv}
```

**Comment by TylerLeonhardt** [comment]
This issue should be fixed in 0.2.0 (Current latest). Keep in mind, the result won't look exactly like the old out-gridview because the views can only be accessed _within_ PowerShell but at least you shouldn't have this error.
